sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1835,6 +1933,9 @@ pi_result piPluginInit(pi_plugin *PluginInit) {
   PiESimdDeviceAccess->data =
       reinterpret_cast<void *>(new sycl::detail::ESIMDDeviceInterface());
 
+  // Surface Index 254 is pre-defined surface index for SLM
+  (*PiESimdSurfaceMap)[254 /* __ESIMD_DNS::SLM_BTI */] = nullptr;
```

Comment:
254 should be replaced with a named constant

Reply:
Will update. I was confused with the location of its definition in a `common.hpp` - there are three `common.hpp` in `sycl/include`.